### ISSUE TITLE
Update partials/navbar.html to use variable for header icon

### ISSUE THREAD

**Issue by shah** [adobe/hugo-spectrum#15]
This is a great theme and I'm enjoying using it. One thing that seems to be hard-coded is the icon in the header and I was curious if it possible to change the following:

```html
<a is="coral-shell-homeanchor" icon="AdobeLogo" href="{{ .Site.Home.RelPermalink }}">{{ .Site.Title }}</a>
```

to

```html
<a is="coral-shell-homeanchor" icon="{{ .Site.HeaderIcon }}" href="{{ .Site.Home.RelPermalink }}">{{ .Site.Title }}</a>
```

The actual variable `{{ .Site.HeaderIcon }}` could be whatever makes sense to the developers. Any chance we could add some configurability to the navigation bar? Thanks!

**Comment by alexander-falca** [comment]
@shah I think we can add this configurability to the navigation bar.

**Comment by shah** [comment]
thanks @alexander-falca - if it helps, here are two suggestions in my forked repo:

https://github.com/adobe/hugo-spectrum/compare/master...netspective-studios:master

* Adding the variable for header icon
* Adding a new unsafe-html shortcode so that I can use all the nice `<coral-*>` HTML components directly in Markdown

Please let me know if a PR is helpful or you can use the suggestions as ideas and I'll keep using my forked repo until your upstream updates are done.

**Comment by amuraru** [comment]
Fixed by #16
